packages/core/elasticsearch/core-elasticsearch-client-server-internal/src/log_query_and_deprecation.ts

```diff
@@ -171,6 +171,16 @@ function getQueryMessage(
   }
 }
 
+function getResponseSizeExceededErrorMessage(error: errors.RequestAbortedError): string {
```

Comment:
Not sure I spotted it (mobile review), will this log something we can easily grep for?

Reply:
Yeah, it has a dedicated logger context